clang/lib/Parse/ParseDeclCXX.cpp

```diff
@@ -100,7 +103,7 @@ Parser::DeclGroupPtrTy Parser::ParseNamespace(DeclaratorContext Context,
     } while (MoreToParse);
   };
 
-  ReadAttributes();
+  ReadAttributes(/*TrailingAttrs*/ false);
```

Comment:
I would just call `CheckProhibitedCXX11Attribute` here

Reply:
@cor3ntin thanks for the review. I've changed param

Reply:
Sorry for the misunderstanding. `CheckProhibitedCXX11Attribute` is an existing function that emits an error on missplaced attributes

Reply:
@cor3ntin is it acceptable to use `DiagnoseAndSkipCXX11Attributes` to reduce errors list?

Reply:
Yep, that seems reasonable